Sources/MapboxNavigation/StatusView.swift

```diff
@@ -46,6 +46,39 @@ public class StatusView: UIControl {
         }
     }
     
+    var statuses: [Status] = []
+
+    /**
+    `Status` is a struct which stores information to be displayed by the `StatusView`
+     */
+    public struct Status {
+        public var identifier: String
+        let title: String
+        var spinner: Bool = false
+        let duration: TimeInterval
+        var animated: Bool = true
+        var interactive: Bool = false
+        var priority: Priority
+    }
+    
+    /**
+     `Priority` is used to display `Status`es by importance
+     */
+    public struct Priority: RawRepresentable {
+        public typealias RawValue = Int
+
+        public var rawValue: Int
+
+        public init(rawValue: Int) {
+            self.rawValue = rawValue
+        }
+//        — Highest Priority —
+//            rerouting (rawValue = 0)
+//            enable precise location (rawValue = 1)
+//            simulation banner (rawValue = 2)
+//        — Lowest Priority —
```

Comment:
The documentation comment for Priority should indicate that lower values are higher priority.